### PR TITLE
added ruby

### PR DESCRIPTION
**Pull request by lkillgore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ These derived images include a set of standard capabilities that enable many of
 - Node.js 8.10.0 LTS
 - Powershell Core v6.0.1
 - Python 2.7.12 and 3.5.2
+- Ruby 2.3.1
```
Comment (madhurig) [inline README.md]: Why did we choose this version? Aren't there more versions of Ruby we want on these images? Should we install or add to tools cache?

Comment (lkillgore) [reply README.md]: This is the one that is installed by apt-get.  There is supposedly a way to install different versions, but I have not found it to be successful.  I think the tools cache mechanism is probably the better (and more stable) way to install multiple ruby versions.

File: ubuntu/standard/Dockerfile.template [hunk 2]
```diff
@@ -140,6 +140,11 @@ RUN apt-get update \
     python3-pip \
  && rm -rf /var/lib/apt/lists/*
 
+# Install Ruby
+RUN apt-get update \
+  && apt-get install -y ruby ruby-dev ruby-bundler \
```
Comment (stepro) [inline ubuntu/standard/Dockerfile.template]: nit: remove one space at the beginning of this line and the next so the commands are aligned.

Comment (lkillgore) [reply ubuntu/standard/Dockerfile.template]: 👍

File: README.md (hunk 1 above)
Comment (stepro) [reply README.md]: One thing to watch out for with apt-get is that you might get different versions on 14.04 vs. 16.04. Indeed for Ruby, I'm seeing this:

- Ubuntu 14.04 gets ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]
- Ubuntu 16.04 gets ruby 2.3.1p112 (2016-04-26) [x86_64-linux-gnu]

So we should update the README like "Ruby 1.9.3 (14.04) or 2.3.1 (16.04)".

Comment (lkillgore) [reply README.md]: ah!  Thanks for letting me know!

**Comment by stepro** [comment]
There is a freeze on pushing of new images at the moment, but I will be building and pushing an updated image at the earliest opportunity.

**Comment by lkillgore** [comment]
Thanks!

**Comment by stepro** [comment]
The images have been updated.

**Pull request merged**